x-pack/test/apm_api_integration/tests/transactions/transactions_groups_comparison_statistics.ts

```diff
@@ -111,19 +124,30 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         expect(response.status).to.be(200);
 
-        const transactionsGroupsComparisonStatistics = response.body as TransactionsGroupsComparisonStatistics;
+        const {
+          currentPeriod,
+          previousPeriod,
+        } = response.body as TransactionsGroupsComparisonStatistics;
 
-        expect(Object.keys(transactionsGroupsComparisonStatistics).length).to.be.eql(
-          transactionNames.length
-        );
+        expect(Object.keys(currentPeriod)).to.be.eql(transactionNames);
+
+        const currentPeriodItems = Object.values(currentPeriod).map((data) => data);
+        const previousPeriodItems = Object.values(previousPeriod).map((data) => data);
```

Comment:
What's the point of this map?

Reply:
Just did what you suggested here, https://github.com/elastic/kibana/pull/91435#discussion_r581038144, or did I miss something?

Reply:
Object.values() is enough no?

Reply:
> Object.values() is enough no?

hahaha yes, it is.